client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -181,6 +184,7 @@ const PluginsBrowserListElement = ( props ) => {
 };
 
 const InstalledInOrPricing = ( {
+	activePlugins,
```

Comment:
There is a reason to receive this value as a property instead of selecting and calculating direct at this component?

This would avoid all this prop drilling and would guarantee all the components using `PluginBrowserItem` will show the `active / inactive` tag with zero effort.

Reply:
Sounds like a worthwhile improvement. Nice catch.

Reply:
> There is a reason to receive this value as a property instead of selecting and calculating direct at this component?

I think I have a deeply routed philosophy on dividing smart and presentational components and use props or context to pass down props depending on the tree deepness and thats why I tried to do any kind of selecting or fetching on what I thought was the smart component `PluginsBrowser`

> This would avoid all this prop drilling and would guarantee all the components using PluginBrowserItem will show the active / inactive tag with zero effort.

I was mostly trying to keep the API of this component as deterministic as possible just in case we want to reuse it somehow but it totally make senses and looks like a worthwhile improvement and will probably have no major impact over tests, simplify code and fix the prop drilling issue (and totally makes sense since we are just using `context` to fix that prop-drilling issue). Awesone catch!